core/src/main/cpp/gdn/projection_mesh/projection_mesh_pool.cpp

```diff
@@ -0,0 +1,23 @@
+#include "projection_mesh_pool.h"
+
+namespace gast {
+
+ProjectionMeshPool::ProjectionMeshPool() = default;
+
+ProjectionMeshPool::~ProjectionMeshPool() = default;
```

Comment:
Seems like the destructor should be releasing the cached resources (if any has been created).

Reply:
Done.